ydb/public/sdk/cpp/include/ydb-cpp-sdk/client/topic/control_plane.h

```diff
@@ -42,13 +42,15 @@ class TConsumer {
 
     const std::string& GetConsumerName() const;
     bool GetImportant() const;
+    TDuration GetAvailabilityPeriod() const;
```

Comment:
От TDuration и TInstant в прекрасном будущем хотелось бы избавиться, может лучше std::chrono::duration тут сделаем?

Reply:
Смешивать и интерфейсе разные стили сильно не хочется. Соседние по смыслу функции возвращают TDuration.
Мы за батчёвый переезд на chrono.
```suggestion
    TDuration GetAvailabilityPeriod() const;
```